tests/unittests/test_exception_handling.py

```diff
@@ -0,0 +1,125 @@
+import tap_klaviyo.utils as utils_
```

Comment:
@harshpatel4crest Did you remove the same file from "tests/" folder?

Reply:
done